spec/unit/pops/validator/validator_spec.rb

```diff
@@ -107,6 +107,15 @@ def validate(model)
         SOURCE
         expect(validate(parse(source))).to have_issue(Puppet::Pops::Issues::IDEM_NOT_ALLOWED_LAST)
       end
+
+      it "detects a resource declared without title in #{type}" do
+        source = <<-SOURCE
+          #{type} nope {
+            notify { message => 'Nope' }
+          }
+        SOURCE
+        expect(validate(parse(source))).to have_issue(Puppet::Pops::Issues::RESOURCE_WITHOUT_TITLE)
+      end
```

Comment:
I want to see additional test cases with epressions before, before and after, and after the faulty expression.